docs/contributor/count-contributions.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Ensure your contributions are counted
+---
+
+## Sign The Contributor License Agreement
+If you already contribute some PRs to [karmada-io](https://github.com/karmada-io), the CLA is signed.
```

Comment:
Contribution is no more than PRs, also including issues and comments.

Reply:
sign CLA only needed when createing PRs. issues and coments don't need it.

Reply:
How about moving to ## Check Your contributions in the dashboard?